multicluster/controllers/multicluster/serviceexport_controller.go

```diff
@@ -378,12 +383,21 @@ func (r *ServiceExportReconciler) handleEndpointDeleteEvent(ctx context.Context,
 	return nil
 }
 
+func (r *ServiceExportReconciler) updateSvcExportAnnotation(svcExport k8smcsv1alpha1.ServiceExport) error {
```

Comment:
nit: this should use pointer, otherwise there would be an unnecessary copy.

Reply:
done